overlord/patch/patch.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 // Level is the current implemented patch level of the state format and content.
-var Level = 6
+var Level = 7
```

Comment:
That's bogus. Needs to be adapted to be a sublevel after the follow up lands.

Reply:
The new patch also needs to be reviewed to ensure it's idempotent. In other words, it needs to work properly even when it goes back and forth between 6.0 and 6.1 several times.

Reply:
Yes, sure, this was implemented before I learned we have hard barrier preventing rollbacks. Will add tests as well, thanks!